### ISSUE TITLE
[BUG]  avoid-use-expanded-as-spacer causing warning in analyze command

### ISSUE THREAD

**Issue by vitoramaral10** [dart-code-checker/dart-code-metrics#1070]
* **Dart code metrics version:** 5.0.1
* **Dart sdk version:** 2.18.4

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
# This file configures the analyzer, which statically analyzes Dart code to
# check for errors, warnings, and lints.
#
# The issues identified by the analyzer are surfaced in the UI of Dart-enabled
# IDEs (https://dart.dev/tools#ides-and-editors). The analyzer can also be
# invoked from the command line by running `flutter analyze`.

# The following line activates a set of recommended lints for Flutter apps,
# packages, and plugins designed to encourage good coding practices.
include: package:flutter_lints/flutter.yaml


analyzer:
  plugins:
    - dart_code_metrics
 

linter:
  # The lint rules applied to this project can be customized in the
  # section below to disable rules from the `package:flutter_lints/flutter.yaml`
  # included above or to enable additional rules. A list of all available lints
  # and their documentation is published at
  # https://dart-lang.github.io/linter/lints/index.html.
  #
  # Instead of disabling a lint rule for the entire project in the
  # section below, it can also be suppressed for a single line of code
  # or a specific dart file by using the `// ignore: name_of_lint` and
  # `// ignore_for_file: name_of_lint` syntax on the line or in the file
  # producing the lint.
  rules:
    # avoid_print: false  # Uncomment to disable the `avoid_print` rule
    # prefer_single_quotes: true  # Uncomment to enable the `prefer_single_quotes` rule

# Additional information about this file can be found at
# https://dart.dev/guides/language/analysis-options
dart_code_metrics:
  anti-patterns:
    - long-method
    - long-parameter-list
  metrics:
    cyclomatic-complexity: 20
    maximum-nesting-level: 5
    number-of-parameters: 4
    source-lines-of-code: 50
    lines-of-executable-code: 50
  metrics-exclude:
    - test/**
  rules-exclude:
    - lib/generated_plugin_registrant.dart
  rules:
    #Common code
    - avoid-banned-imports
    - avoid-border-all
    - avoid-cascade-after-if-null
    - avoid-collection-methods-with-unrelated-types
    - avoid-duplicate-exports
    - avoid-dynamic
    - avoid-global-state
    - avoid-missing-enum-constant-in-map
    - avoid-nested-conditional-expressions:
      acceptable-level: 2
    - avoid-non-ascii-symbols
    - avoid-redundant-async 
    - avoid-returning-widgets
    - avoid-throw-in-catch-block
    - avoid-unnecessary-type-assertions
    - avoid-unnecessary-type-casts
    - avoid-unrelated-type-assertions
    - avoid-unused-parameters
    - binary-expression-operand-order
    - check-for-equals-in-render-object-setters
    - consistent-update-render-object
    - double-literal-format
    - format-comment
    - member-ordering:
      alphabetize: true
    - member-ordering
    - newline-before-return
    - no-boolean-literal-compare
    - no-empty-block
    - no-equal-then-else
    - no-object-declaration
    - prefer-async-await
    - prefer-commenting-analyzer-ignores
    - prefer-conditional-expressions
    - prefer-correct-edge-insets-constructor
    - prefer-correct-identifier-length:
      exceptions: [ 'z' ]
      max-identifier-length: 30
      min-identifier-length: 4
    - prefer-correct-test-file-name
    - prefer-correct-type-name
    - prefer-enums-by-name
    - prefer-first
    - prefer-immediate-return
    - prefer-iterable-of
    - prefer-last
    - prefer-match-file-name
    - prefer-trailing-comma
    #Flutter specific
    - always-remove-listener
    - avoid-border-all
    - avoid-returning-widgets
    - avoid-unnecessary-setstate
    - avoid-wrapping-in-padding
    - avoid-use-expanded-as-spacer
    - prefer-const-border-radius
    - prefer-correct-edge-insets-constructor
    - prefer-single-widget-per-file

```

</details>

**What did you do? Please include the source code example causing the issue.**

```sh
 flutter pub run dart_code_metrics:metrics analyze lib
```

**What did you expect to happen?**
```sh
 Analysis is completed. Preparing the results: 5.9s
```
**What actually happened?**
```sh
$  flutter pub run dart_code_metrics:metrics analyze lib
✔ Analysis is completed. Preparing the results: 5.9s

analysis_options.yaml:
WARNING 'avoid-use-expanded-as-spacer' is not recognized as a valid rule name.
        at C:\*****\analysis_options.yaml:106:7
        unknown-config : https://dartcodemetrics.dev/docs/rules


pub finished with exit code 2
```
**Are you willing to submit a pull request to fix this bug?**
No

**Comment by incendial** [comment]
@vitoramaral10 thanks, the docs are wrong, the rule name is `avoid-expanded-as-spacer`. Please use it instead of `avoid-use-expanded-as-spacer`.

**Comment by vitoramaral10** [comment]
Thanks, this solution works for me!

```sh
$  flutter pub run dart_code_metrics:metrics analyze lib
✔ Analysis is completed. Preparing the results: 5.6s

✔ no issues found!
```
